FWCore/Common/interface/TriggerResultsByName.h

```diff
@@ -46,11 +46,8 @@ namespace edm {
   class HLTPathStatus;
 
   class TriggerResultsByName {
-
   public:
-
-    TriggerResultsByName(TriggerResults const* triggerResults,
-                         TriggerNames const* triggerNames);
+    TriggerResultsByName(TriggerResults const* triggerResults, TriggerNames const* triggerNames);
```

Comment:
This time it put all arguments on the same line. What is the threshold for same line versus different lines?